components/schemas/environments/services/lb/telemetry/LoadBalancerTelemetryUrlMetrics.yml

```diff
@@ -14,15 +14,57 @@ properties:
           nullable: true
           type: object
           required:
+            - host
+            - method
+            - path
             - total
+            - last_hit
+            - handlers
           properties:
+            host:
+              type: string
+              description: The host name of the URL request
+              example: abc.cycle.io
+            method:
+              type: string
+              description: The method of the UrRL request
+              example: GET
+            path:
+              type: string
+              description: The path name of the URL request
+              example: /pathname
             total:
               type: integer
+              description: The number of requests to a certain URL
+              example: 11
+            last_hit:
+              description: The timestamp of the most recent hit to the URL
```

Comment:
Don't use the word timestamp here since it is NOT a timestamp, it's an ISO datetime. I would just say "the date of the most recent hit.."